content/fa/community/code-of-conduct.md

```diff
@@ -0,0 +1,19 @@
+---
+title: آیین‌نامه رفتاری انجمن کوبرنتیز
+body_class: code-of-conduct
+cid: code-of-conduct
+---
+
+_کوبرنتیز از 
+[آیین‌نامه رفتاری CNCF](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) پیروی می‌کند.
+متن آیین‌نامه رفتاری CNCF در زیر آمده است
+[commit 71412bb02](https://github.com/cncf/foundation/blob/71412bb029090d42ecbeadb39374a337bfb48a9c/code-of-conduct.md)._
+
+---
```

Comment:
```suggestion
<div id="cncf-code-of-conduct">
{{< include "static/cncf-code-of-conduct.md" >}}
</div>

---
```

@xirehat I just wanted to check if skipping this section was intentional. :)
I think it should eventually be included.